src/background.js

```diff
@@ -83,14 +85,23 @@ const idForItem = item => `${item.time}-${item.url}`;
 
 const messageOps = {
   schedule: message => {
+    Metrics.scheduleSnoozedTab(message);
     const toSave = {};
     toSave[idForItem(message)] = message;
     return browser.storage.local.set(toSave).then(updateWakeAlarm);
   },
-  cancel: message =>
-    browser.storage.local.remove(idForItem(message)).then(updateWakeAlarm),
+  cancel: message => {
+    Metrics.cancelSnoozedTab(message);
+    return browser.storage.local.remove(idForItem(message)).then(updateWakeAlarm);
+  },
+  click: message => {
+    Metrics.clickSnoozedTab(message);
+  },
   update: message =>
-    messageOps.cancel(message.old).then(() => messageOps.schedule(message.updated))
+    messageOps.cancel(message.old).then(() => messageOps.schedule(message.updated)),
```

Comment:
Not sure - updates weren't listed as one of the measurements in [the plan so far](https://github.com/bwinton/SnoozeTabs/issues/45). Seems like a handy thing to measure, though